admin/python/lsst/qserv/admin/partConfig.py

```diff
@@ -109,6 +109,14 @@ def isView(self):
         """Returns True if table is a view"""
         return bool(self.data.get('view', False))
 
+    def isDirector(self, tableName):
+        """
+        Returns True if table is a director table. Director table name
+        is determined by dirTable parameter, if dirTable is not set then
+        Object is assumed to be a director table.
+        """
+        return tableName == self.get('dirTable', 'Object')
```

Comment:
I would like not to have default value for dirTable, this is an important parameter so I think it should always be explicited in cfg files.